src/App.tsx

```diff
@@ -1,61 +1,94 @@
+import { FC, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './api/todos';
+import { TodoList } from './components/TodoList';
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+  const [titleValue, setTitleValue] = useState<string>('');
+  const [selectedUser, setSelectedUser] = useState<number>(0);
+  const [isClicked, setIsClicked] = useState<boolean>(false);
```

Comment:
It's better to create a separate state for errors instead of just tracking click.